src/util.js

```diff
@@ -159,6 +160,13 @@ class Util {
       return object;
     return Promise.resolve(object);
   }
+
```

Comment:
Why not using https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID ?
Or could you describe your algorithm and the reasons to use this?

PS: I'd like to have as little external dependencies as possible in the SDK, so it's good if we can avoid one